app/ng-ui-widgets-category/tabs/styling/article.md

```diff
@@ -1,3 +1,7 @@
 > **Note:** - The integration with `nativescript-theme` and the support for custom CSS is currently under development and is on its way.
 
-<snippet id='tabs-theming-css-ng'/>
\ No newline at end of file
+<snippet id='tabs-theming-css-ng'/>
+
+> **Note:** Currently, we can set only the `backgroundColor`, `color`, `fontFamily`, `fontSize`, `fontStyle`, `fontWeight` and `textTransform` CSS properties to the `Label` and `Image` components inside the TabStripItem. More about the usage of those CSS properties can be found in the [Styling]({%slug styling%}#supported-css-properties) article.
```

Comment:
Reword this, so that it's clear that these are simply properties and point to the Styling article for usage as CSS properties.